docs/modal.html

```diff
@@ -0,0 +1,72 @@
+<html>
+    <head>
+        <title>MarvelUI</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+        <link rel="stylesheet" href="./stylesheets/main.css"/>
+        <link rel="stylesheet" href="./custom-stylesheets/styles.css"/>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+    </head>
+    <body>
+        <aside>
+            <div class="logo-container">
+                <a href="./index.html"><img class="logo" src="./assets/logo.svg" alt="logo"></a>
+            </div>
+            <nav>
+                <ul>
+                    <li><a href="./alert.html">Alert</a></li>
+                    <li><a href="./avatar.html">Avatar</a></li>
+                    <li><a href="./badge.html">Badge</a></li>
+                    <li><a href="./button.html">Button</a></li>
+                    <li><a href="./card.html">Card</a></li>
+                    <li><a href="./image.html">Image</a></li>
+                    <li><a href="./input.html">Input</a></li>
+                    <li><a href="./modal.html">Modal</a></li>
+                </ul>
+            </nav>
+        </aside>
+        <main>
+            <div class="main-section">
+                <div class="main-container flex-col-wrap-center">
+                    <h1 class="header">Modals</h1>
+                    <p class="sub-header">Modals inform users about a task and can contain critical information, require decisions, or involve multiple tasks.</p>
+                    <div class="modal-container flex-col-wrap-center">
+                        <h2>Alert Modal</h2>
+                        <div class="flex-row-wrap-center">
+                            <div class="modal-wrapper">
+                                <div class="modal-header">Modal header</div>
+                                <div class="modal-body">
+                                    Modal body text
+                                    <div class="modal-actions">
+                                        <button class="modal-action">ACTION 1</button>
+                                        <button class="modal-action">ACTION 2</button>
```

Comment:
You can make use of your primary and secondary buttons here instead of just `modal-action` class

Reply:
Sure. I will try it out. Thanks for approval.